util/container/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#---------------------
+# Ubuntu 22.04 base
+#---------------------
+FROM ubuntu:22.04
+
+#---------------------
+# Install necessary packages
+#---------------------
+RUN apt-get -y update
+RUN apt-get -y install git
+RUN apt-get -y install autoconf help2man perl python3 make flex bison g++ 
+RUN apt-get -y install libfl2  libfl-dev
+RUN apt-get -y install python3 pip
+
+#---------------------
+# Compile verilator from source code
+#---------------------
+RUN git clone https://github.com/verilator/verilator
+WORKDIR verilator/
+RUN git checkout v5.006
+ENV VERILATOR_ROOT=/verilator
+RUN autoconf
+RUN ./configure
+RUN make -j `nproc`
+RUN make test
+ENV PATH="$VERILATOR_ROOT/bin:$PATH"
```

Comment:
Since building verilator is the most time-consuming step, I would put it last in the container build process.
If other things fail, then the docker build will fail faster, which is better.

Reply:
@JosseVanDelm hmm. Isn't it better if it was first? When I was building the container, I realized that if I change a line before this, it re-runs from the changed line and runs this part again. I was thinking that the Verilator builds first so that changes, shorter tasks don't need to re-run the Verilator build again?

Reply:
mmm yes I agree!